app/src/main/res/layout/history_metadata_header.xml

```diff
@@ -6,29 +6,36 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="56dp"
+    android:layout_height="wrap_content"
     android:layout_marginTop="40dp">
 
     <androidx.appcompat.widget.AppCompatTextView
-        style="@style/Header20TextStyle"
         android:id="@+id/header"
+        style="@style/Header16TextStyle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/history_metadata_header"
-        android:layout_marginVertical="16dp"
+        android:paddingVertical="16dp"
         android:maxLines="2"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
     <com.google.android.material.button.MaterialButton
         android:id="@+id/show_all_button"
-        style="@style/Button12TextStyle"
+        style="@style/Button14TextStyle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:background="?android:attr/selectableItemBackground"
         android:text="@string/recent_tabs_show_all"
         android:textColor="@color/home_show_all_button_text"
+        android:contentDescription="@string/past_explorations_show_all_content_description"
+        android:paddingVertical="16dp"
+        android:paddingStart="16dp"
+        android:paddingEnd="0dp"
+        android:letterSpacing="-0.0024"
```

Comment:
I think this is an artificial of looking at those iOS designs letterspacing spec. I don't think we have any for Android.

Reply:
Removing that as we speak ;)